docs/source/changelog.md

```diff
@@ -6,7 +6,7 @@
 
 ## 7.0
 
-### [7.0.0] - 2024-10-21
+### [7.0.0] - 2024-10-25
 
 ([full changelog](https://github.com/jupyterhub/kubespawner/compare/6.2.0...7.0.0))
```

Comment:
We had kubespawner upgrade docs, lets link to them here as well.

Lgtm otherwise, i'm walking about with family atm so if someone wants please feel free to finalize this and do a release

Reply:
yes, trying to figure out how I lost them. I'll put it back. Thanks for noticing!

Reply:
oh, I didn't lose them. They are right below in the breaking changes. So the link is there and near the top. I think that is sufficient. I'll go ahead and make the release.